src/main/java/io/oneko/kubernetes/impl/DeploymentManagerImpl.java

```diff
@@ -110,6 +110,8 @@ public ReadableProjectVersion deploy(final WritableProjectVersion version) {
 					helmCommands.uninstall(deployment.getReleaseNames());
 				}
 
+				Thread.sleep(10000);
```

Comment:
Won't this also block the HTTP request sent by the frontend?

And shouldn't this be in the block above, so it's only executed when an application is actually uninstalled?